OpenLiberty.MD

```diff
@@ -0,0 +1,156 @@
+2018-10-10 13:51:41.683  INFO 14858 --- [cutor-thread-10] o.s.c.support.DefaultLifecycleProcessor  : Starting beans in phase 0
+2018-10-10 13:51:41.771  INFO 14858 --- [cutor-thread-10] inventory.Application                    : Started Application in 9.968 seconds (JVM running for 14.676)
+Inventory microservice is ready for business...
+Will not use Message Hub due to invalid configuration
+[AUDIT   ] CWWKZ0001I: Application thin-liberty-inventory-spring-0.1-SNAPSHOT started in 10.874 seconds.
+[AUDIT   ] CWWKF0012I: The server installed the following features: [servlet-4.0, springBoot-1.5].
+[AUDIT   ] CWWKF0011I: The server BoostServer is ready to run a smarter planet.
+```
+
+4. Validate. You should get a list of all inventory items:
+```bash
+$ curl http://localhost:8080/micro/inventory
+```
+
+That's it, you have successfully deployed and tested the Inventory microservice running on Spring Boot using Open Liberty server.
+
+## Deploy the Inventory Docker Container
+
+In this section you will run the Spring Boot application on Open Liberty using Docker. Before we show you how to do so, you will need to deploy a MySQL Docker container and populate it with data as shown in the [Deploy the MySQL Docker Container](#deploy-the-mysql-docker-container) and [Populate the MySQL Database](#populate-the-mysql-database) sections, respectively.
+
+Once MySQL is ready and populated, we can run the Spring Boot Inventory application on OpenLiberty dcoker as follows:
+
+1. Open [`src/main/resources/application.yml`](src/main/resources/application.yml) file, enter the following values for the fields under `spring.datasource`, and save the file:
+    * **url:** jdbc:mysql://inventorymysql:3306/inventorydb?useSSL=false
+    * **username:** dbuser
+    * **password:** password
+    * **port:** 3306
+
+2. Modify your pom to generate the docker image. Follow the below steps.
+- Open [`pom.xml`](pom.xml)
+- Modify the `boost-maven-plugin` as below.
+
+```
+            <plugin>
+                <groupId>io.openliberty.boost</groupId>
+                <artifactId>boost-maven-plugin</artifactId>
+                <version>0.1</version>
+                <executions>
+                  <execution>
+                    <!-- <phase>package</phase>
+                    <goals>
+                        <goal>package</goal>
+                    </goals> -->
+                    <goals>
+                      <goal>docker-build</goal>
+                    </goals>
+                  </execution>
+                </executions>
+            </plugin>
+```
+
+3. To deploy the Inventory container, run the following commands:
+
+Before building the Docker image, make sure there are no other dockerfiles in your repo. If a dockerfile already exists, please remove it.
+
+```bash
+# Build the Docker Image
+$ mvn clean install 
+
+Once done, a docker image named `liberty-inventory-spring:latest` will be built automatically for you.
+
+# Start the Inventory Container
```

Comment:
Missing the opening triple-backtick here.

Reply:
Corrected the format